### PR TITLE
Validation errors

### PR DESCRIPTION
**Pull request by Favizille**

### CODE REVIEW

**Comment by elishaukpong** [comment]
resolve conflicts @Favizille

File: app/Http/Controllers/LoginController.php [hunk 1]
```diff
@@ -12,11 +12,10 @@ public function __construct(private LoginRepository $loginRepository)
 
     public function login(LoginRequest $request)
     {
+        $errors = ["Login Credentials are invalid"];
```
Comment (elishaukpong) [inline app/Http/Controllers/LoginController.php:15]: avoid temporary variables please, if you wont use a variable in more than one place, don't create it. 

Unless it adds meaning to a complex logic which in this case doesn't.

`->withErrors(["Login Credentials are invalid"]);` is better

File: app/Http/Controllers/LoginController.php [hunk 2]
```diff
@@ -12,11 +12,10 @@ public function __construct(private LoginRepository $loginRepository)
...
+
         if(! $this->loginRepository->login($request->validated()) ) {
             return redirect()->route('login')
-                ->with("message", "Login failed");
-        }
-
-        return redirect()->route('dashboard.index');
```
Comment (elishaukpong) [inline app/Http/Controllers/LoginController.php:20]: why are you removing this redirect to dashboard?

File: resources/views/login.blade.php [hunk 3]
```diff
@@ -25,6 +25,15 @@
               <p class="mb-6">Please enter your user information.</p>
             </div>
 
+            @if(count($errors) > 0)
+                @foreach( $errors->all() as $message )
+                    <div class="alert alert-danger display-hide">
+                    {{-- <button class="close" data-close="alert"></button> --}}
```
Comment (elishaukpong) [inline resources/views/login.blade.php:31]: dont push any commented out code, if you're not using a code - delete it.

File: resources/views/login.blade.php [hunk 4]
```diff
@@ -25,6 +25,15 @@
...
+                    <span>{{ $message }}</span>
+                    </div>
+                @endforeach
+            @endif
+
             <form action="{{route("login")}}" method="POST">
```
Comment (elishaukpong) [inline resources/views/login.blade.php:37]: have space between `{{ route('login) }}`

**Review by elishaukpong — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/LoginController.php (hunk 2 above)
Comment (Favizille) [reply app/Http/Controllers/LoginController.php:20]: I am not removing it, it's still in the code

**Pull request merged**